paasta_tools/deployd/master.py

```diff
@@ -245,6 +247,7 @@ def prioritise_bouncing_services(self):
             self.inbox_q.put(ServiceInstance(
                 service=service,
                 instance=instance,
+                cluster=self.config.get_cluster(),
                 watcher=self.__class__.__name__,
                 bounce_by=int(time.time()),
                 bounce_timers=None,
```

Comment:
`self.config` here is the public config not the config of the service instance.

I've left the `InboxQueue` alone for now. We're basically getting two "priority mechanisms", the `bounce_by` which is used in the `InboxQueue` and the `bounce_priority` for the `BounceQueue`. Long term we might want to pull the concept of the InboxQueue, or limit it to just staggering repeated runs of `setup_marathon_job`